package.json

```diff
@@ -134,12 +135,11 @@
     "prepare": "husky install",
     "start": "wp-scripts start",
     "test": "npm-run-all --parallel test:*",
-    "test:e2e": "wp-scripts test-e2e --config=tests/e2e/jest.config.js",
+    "test:e2e": "cross-env JEST_PUPPETEER_CONFIG=./tests/e2e/puppeteer.config.js wp-scripts test-e2e --config=tests/e2e/jest.config.js",
     "test:e2e:help": "npm run test:e2e -- --help",
     "test:e2e:watch": "npm run test:e2e -- --watch",
     "test:e2e:interactive": "npm run test:e2e -- --puppeteer-interactive",
-    "pretest:e2e:ci": "npx patch-package",
```

Comment:
`patch-package` is no longer needed? Introduced by you in 66b505e7c9827d4b1fad633f7f9a393f76806bf7.

Reply:
Yes. I removed the `patch-package` as a part of https://github.com/ampproject/amp-wp/pull/7641 but forgot to remove this npm script.